src/System.Text.Json/src/System/Text/Json/Node/JsonNumber.cs

```diff
@@ -7,89 +7,357 @@
 
 namespace System.Text.Json
 {
-    public partial class JsonNumber : JsonNode
+    public partial class JsonNumber : JsonNode, IEquatable<JsonNumber>
```

Comment:
Presumably we are defering API documentation (`/// xml comments` like summary/parameters/returns/exceptions) until after the API is more stable. Is the plan to add them once the type/impl/tests moves to corefx?

Reply:
I'll add documentation in main corefx PR.